src/ts/component/editor/page.tsx

```diff
@@ -567,19 +567,25 @@ const EditorPage = observer(class EditorPage extends React.Component<Props, Stat
 		const readonly = this.isReadonly();
 		const styleParam = this.getStyleParam();
 
+		let ret = false;
```

Comment:
I'd put a comment what this value is. For me as developer who doesn't know this repo code this variable means nothing.